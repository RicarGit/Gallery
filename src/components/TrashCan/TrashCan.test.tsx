import { render } from '@testing-library/react'
import { TrashCan } from '.'

describe('PhotoCard', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <TrashCan name='photo name' photos={[]} setPhotos={() => []} />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})