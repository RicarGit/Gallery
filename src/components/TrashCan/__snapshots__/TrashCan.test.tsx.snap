// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoCard should match snapshot 1`] = `
<div
  class="styles__TrashCan-sc-195alnh-0 iiGOar"
>
  <img
    alt="trash icon"
    class="trashCan"
    src="delete.svg"
  />
</div>
`;
